docs/api-reference-1/wordpress-source.md

```diff
@@ -483,19 +485,16 @@ For more info, visit the [WP REST API reference](https://developer.wordpress.org
 const { api } = libraries.source;
 
 // Get posts from categories 2, 3 and 4
-api.get({ endpoint: "posts", params: { _embed: true, categories: '2,3,4' } })
-  .then(dataRequested => { /* do something with dataRequested */ })
+const postsCategories = await api.get({ endpoint: "posts", params: { _embed: true, categories: '2,3,4' } })
 
 // Get the page 14
-api.get({ endpoint: "pages", params: { _embed: true, include: '14' } });
-  .then(dataRequested => { /* do something with dataRequested */ })
+const page14 = await api.get({ endpoint: "pages", params: { _embed: true, include: '14' } });
 
 // Other endpoints: 
-api.get({ 
+const postBeautiesGullfoss = await api.get({ 
   endpoint: "/frontity/v1/discovery",
```

Comment:
This endpoint doesn't exist anymore because it belongs to our old plugin which is now deprecated. I don't know why this example is here, I guess we added it at the very beginning.

I would use another example, something that is used today. For example, the `/acf/v3/posts` endpoint.